rotkehlchen/chain/ethereum/adex/utils.py

```diff
@@ -166,6 +167,7 @@ def deserialize_adex_event_from_db(
             pool_id=pool_id,
             channel_id=cast(str, event_tuple[12]),
             token=token,
+            log_index=cast(int, event_tuple[14]),
```

Comment:
why is the cast needed here?

Reply:
Cause `log_index` is optional and unless I assert one by one they are not None (I do it in a loop) it shows this:
```shell
rotkehlchen/chain/ethereum/adex/utils.py:170: error: Argument "log_index" to "ChannelWithdraw" has incompatible type "Optional[int]"; expected "int"
Found 1 error in 1 file (checked 350 source files)

```

Reply:
In that case this is a limitation of mypy at the moment and for these we never use casts. We do # type: ignore. The reason for that is that when mypy fixes their shit, the mypy compiler actually lets you know that the #type: ignore is unused.

Please change this, and the other casts in the same file. i see there are many.